packages/core/src/styles/_type.scss

```diff
@@ -0,0 +1,17 @@
+@import "@carbon/type/scss/type";
+@import "@carbon/type/scss/font-face/mono";
+@import "@carbon/type/scss/font-face/sans";
+
+@include carbon--type-reset();
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
```

Comment:
Should we be including these here? Or would this be the applications responsibility?

Reply:
I was following the instructions to get started with carbon type, I can remove this if it should be the responsibility of the consumers